ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -691,6 +696,11 @@ stdLowLevelRunNodeArgsIO RunNodeArgs{ rnProtocolInfo } StdRunNodeArgs{..} = do
             (\mc bfc -> bfc { bfcMaxConcurrencyBulkSync = mc })
             srnBfcMaxConcurrencyBulkSync
 
+    disableVersionsAbove :: forall k v. Ord k => k -> Map k v -> Map k v
+    disableVersionsAbove latestNonDevelopmentalVersion =
```

Comment:
I'd call this function `disableVersiosnStrictlyAbove`, at least for me above sometimes is confusing.

Reply:
I changed its semantics a bit, so the name is now more concrete.